### PR TITLE
Handle user with no dates in database

### PR DESCRIPTION
**Pull request by nberard**

### CODE REVIEW

File: includes/database.py [hunk 1]
```diff
@@ -4,27 +4,34 @@
...
         self.insert_count = kwargs.get("insert_count", 1000)
+        self.logger = logger
         self.connection = None
         self.cursor = None
         if auto_connect:
             try:
                 self.connect()
             except psycopg2.OperationalError as e:
-                get_logger().critical('Cannot connect database, error: {msg}'.format(msg=str(e)))
+                self.logger.error('Cannot connect database, error: {msg}'.format(msg=str(e)))
```
Comment (datanel) [inline includes/database.py]: if logger=None this will not work

Comment (nberard) [inline includes/database.py]: i added a safe method to log info but not error yet since logger None is only when adding fixtures in tests

Comment (datanel) [inline includes/database.py]: how `None.error` could work?

Comment (nberard) [inline includes/database.py]: it does not work, it's what i said :p

**Pull request merged**